nixos/modules/system/boot/stratisroot.nix

```diff
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+let
+  types = lib.types;
+in
+{
+  options.boot.stratis = {
+    rootPoolUuid = lib.mkOption {
+      type = types.uniq types.str;
+      description = lib.mdoc ''
+        UUID of the stratis pool that the root fs is located in
+      '';
+      example = "04c68063-90a5-4235-b9dd-6180098a20d9";
+    };
+  };
+  config = {
```

Comment:
This will need to be behind a `mkIf` depending on if stratis is used in the config.

Reply:
done